src/styles/CompareCards.ts

```diff
@@ -66,16 +66,17 @@ export const CompareCardsStyles = (mode: string) => {
           boxShadow: '0px 2px 4px rgba(58, 57, 68, 0.2)',
         },
         '&.content-base': {
+          visibility: 'hidden',
           minHeight: '0',
           height: '0',
```

Comment:
No, this is part of a smooth animation for the dropdowns rather than just appearing and disappearing. I hadn't finished it. But I'll cover this in a separate PR. So let's leave as is for now.